config-model/src/main/java/com/yahoo/vespa/model/container/http/JettyHttpServer.java

```diff
@@ -82,10 +75,17 @@ public void getConfig(ServerConfig.Builder builder) {
 
     private void configureJettyThreadpool(ServerConfig.Builder builder) {
         if (cluster == null) return;
+        if (cluster instanceof ApplicationContainerCluster) {
+            configureApplicationClusterJettyThreadPool(builder);
+        } else {
+            builder.minWorkerThreads(2).maxWorkerThreads(4);
```

Comment:
Have you tried this? I'm pretty sure 2-4 is too low.

Reply:
No, not tried. Will run a few system tests.